mkdocs.yml

```diff
@@ -42,8 +42,10 @@ theme:
   name: 'material'
```

Comment:
Before you merge, please check that the site preview works properly. I can't test this myself, as Gitpod has build erors, and there's no working CodeSpace setup yet.

Reply:
should be working again 🎉